src/components/NotificationsPanel/NotificationsPanel.js

```diff
@@ -1,30 +1,50 @@
 import React from 'react'
 import styled from 'styled-components'
-import { theme, font, BadgeNumber } from '@aragon/ui'
+import { theme, font, observe, redraw, BadgeNumber } from '@aragon/ui'
 import NotificationItem from './NotificationItem'
 
-const NotificationsPanel = ({ notifications = [] }) => (
-  <Main>
-    <Header>
-      <Title>
-        <span>Notifications</span>
-        <BadgeNumber number={4} title={`${4} unread messages`} />
-      </Title>
-      <ActionLink>Clear all</ActionLink>
-    </Header>
-    <Content>
-      {notifications.map(({ date, title, description, unread }, i) => (
-        <NotificationItem
-          key={`${date}${i}`}
-          date={date}
-          title={title}
-          description={description}
-          unread={unread}
-        />
-      ))}
-    </Content>
-  </Main>
-)
+const REDRAW_TIME = 30 * 1000 // refresh dates every 30 sec
```

Comment:
This doesn't work as well as I thought it would, since some of the notifications can still show up in seconds if they're added when the panel's open.

I didn't want to refresh them every second though D:

Reply:
I was thinking — we could instead redraw in each `<NotificationItem />`, where the redraw timers could be automatically adjusted to being every second, every minute, every hour, to etc. as the time difference start to increase.

Reply:
👌 Or a mix of both: one timer every second, minute, hour, which filters the notifications that need to be refreshed by it, depending on their time difference.